app/components/questions/TrafficIntensity.js

```diff
@@ -26,6 +26,26 @@ const CITY_PART_COLOR = {
   "West Side": "#882255",
 };
 
+const TEXT_TRANSLATION = {
+  "English": {
+  "Title": "Traffic Intensity by Area", 
```

Comment:
You can indent the translation parts one more level, that way they are not on the same indentation level as the languages.

Reply:
Actually, this is a code style comment and doesn't need to be blocking.